tools/esdlcmd/esdlcmd_core.cpp

```diff
@@ -1177,7 +1177,7 @@ class Esdl2XmlCmd : public EsdlHelperConvertCmd
     bool optRollup;
 
 public:
-    Esdl2XmlCmd() : optRollup(false)
+    Esdl2XmlCmd() : optRollup(false), optProcessAllIncludes(false)
```

Comment:
For future code, it is generally safer/cleaner to use an initialiser on the field - it is clearer the field is initialised, and it avoids problems if a construct has multiple constructors.

Reply:
OK yes that makes sense. I'll use that style in the future.